helm/values.yaml

```diff
@@ -113,27 +120,48 @@ livenessProbe:
     port: http
   initialDelaySeconds: 5
   failureThreshold: 5
-  periodSeconds: 5
+  periodSeconds: 30
 readinessProbe:
   httpGet:
     path: /healthcheck
     port: http
   initialDelaySeconds: 5
   failureThreshold: 20
-  periodSeconds: 2
+  periodSeconds: 30
 
 # Additional volumes on the output Deployment definition.
-volumes: [ ]
+volumes:
```

Comment:
Why do we need this? I thought istio in ambient mode shouldn't require any changes to deployments